Packs/Securonix/Integrations/Securonix/Securonix.py

```diff
@@ -1666,7 +1669,12 @@ def list_activity_data(client: Client, args) -> tuple[str, dict, dict]:
     from_ = args.get("from", "").strip()
     to_ = args.get("to", "").strip()
     query = escape_spotter_query(args.get("query", "").strip())
-    activity_data = client.list_activity_data_request(from_, to_, query)  # type: ignore
+    max_records = arg_to_number(args.get("max", "1000"), arg_name="max")
```

Comment:
no need to provide the arg name, it could be easily implemented like this:
```suggestion
    max_records = arg_to_number(args.get("max", "1000"))
```